main/main.cpp

```diff
@@ -1585,6 +1585,16 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 #endif
 		}
 
+		uint32_t boot_min_delay = GLOBAL_DEF("application/boot_splash/delay_msec", 0);
+		ProjectSettings::get_singleton()->set_custom_property_info("application/boot_splash/delay_msec",
+				PropertyInfo(Variant::INT,
+						"application/boot_splash/delay_msec",
+						PROPERTY_HINT_RANGE,
+						"0,100,1,or_greater")); // No negative numbers
```

Comment:
I guess it could use `1000` or `2000` as max value, I can imagine that users who would want to ensure a min display time for the splash screen would often want more than 100 ms.

Reply:
It doesn't matter now, because there is `or_greater`. I copied this from here:

https://github.com/godotengine/godot/blob/2214ced7daa92a6eb90bd348475f139a5ef3b959/main/main.cpp#L1352-L1355